frontend/src/pages/modelRegistry/screens/utils.ts

```diff
@@ -79,30 +79,32 @@ export const filterModelVersions = (
   unfilteredModelVersions: ModelVersion[],
   search: string,
   searchType: SearchType,
-): ModelVersion[] =>
-  unfilteredModelVersions.filter((mv: ModelVersion) => {
+): ModelVersion[] => {
+  const searchLower = search.toLowerCase();
+
+  return unfilteredModelVersions.filter((mv: ModelVersion) => {
     if (!search) {
       return true;
     }
 
     switch (searchType) {
-      case SearchType.KEYWORD:
+      case SearchType.KEYWORD: {
```

Comment:
Small nit -- you only need `{` braces around a case when creating a variable inside it to scope it lexically. Other than that it's not needed.

Reply:
I think I added _because_ it was throwing a lexical error when I ran npm run test